src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/instance_hyjal.cpp

```diff
@@ -567,20 +567,20 @@ class instance_hyjal : public InstanceMapScript
         }
 
     protected:
-        int32 trash;
-        uint8 _currentWave;
-        uint8 _bossWave;
-        uint8 _retreat;
-        GuidSet _encounterNPCs;
-        GuidSet _summonedNPCs;
-        GuidSet _baseAlliance;
-        GuidSet _baseHorde;
-        GuidVector _infernalTargets;
-        GuidSet _baseNightElf;
-        GuidSet _ancientGemAlliance;
-        GuidSet _ancientGemHorde;
-        GuidSet _roaringFlameAlliance;
-        GuidSet _roaringFlameHorde;
+        int32 Trash;
```

Comment:
this is wrong, those are private members

https://www.azerothcore.org/wiki/cpp-code-standards#coding-standards

Reply:
They are declared as protected :).

Reply:
yeah yeah and those should be private and not protected

Reply:
ok :)